kaniko-example/README.md

```diff
@@ -30,7 +30,13 @@ This is an example showing how to collect build-info, while using the Kaniko con
 
     ```console
     Run Kaniko:
-    > docker run --rm -v `pwd`:/workspace -v `pwd`/config.json:/kaniko/.docker/config.json:ro gcr.io/kaniko-project/executor:latest --dockerfile=Dockerfile --destination=DOCKER_REG_URL/hello-world:latest --image-name-with-digest-file=image-file-details
+    > IMAGE_TAG=latest
```

Comment:
Unless you run this sequence of commands from within a shell script, the value of  IMAGE_TAG will not be available for the next command.
I therefore suggest changing ```IMAGE_TAG=latest``` to ```export IMAGE_TAG=latest```.